week-1/errors/4.js

```diff
@@ -1,2 +1,5 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+//const 12HourClockTime = "20:53";
+//const 24hourClockTime = "08:53";
+
+const hourClockTime24 = "20:53";
+const hourClockTime12 = "08:53";
```

Comment:
Good work but why did this fix it?

Reply:
The variable names 12HourClockTime and 24hourClockTime are syntactically valid in JavaScript, but they can be misleading if you intend to use them for the specific times you've provided. The values "20:53" and "08:53" are in the 24-hour clock format, not the 12-hour clock format.So If I want to represent these times in a 12-hour clock format, I should use a different variable name to avoid confusion. This way, the variable names better reflect the content they store, and it's clear that hourClockTime24is in the 24-hour clock format. and 12-hour clock format hourClockTime12.